### DISCUSSION TITLE
Pirates can refuel on Republic planets! Is that allowed?

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10622] [Lore]
Okay, not exactly. It was a Bounty Hunter ship I saw doing that. But that begs the question: should Bounty Hunters be allowed to refuel in Republic space?

I find that strange. Why would the Republic allow mercenaries to land on their planets? In fact, why would they allow contract killers to operate in their systems, anyway?

It's quite amusing to be surrounded by Republic ships and still be fired at, with seemingly zero consequences for the aggressors. No one in the Navy questions why there seem to be a bunch of ships attacking the player, a law-abiding Republic citizen? Nobody bothers to call for, or enforce, a temporary ceasefire?

It would make more sense if Bounty Hunter ships were of the hit-and-run type, having powerful single-use weapons to disable the player, because they would have small windows of time between Republic patrols in order to kill the mark.

They would also have to prevent or somehow avoid nearby merchant ships just reviving the player.

This kind of mission would be more fun, I think. The player would have to dodge overwhelmingly powerful projectiles, and even duck their way in and out of various systems, hoping and praying they find a Republic patrol to ward away the bounty hunters.

Or maybe they try to look for merchant fleets to distract the hunters. If you're a mercenary and you see an opportunity to make some really easy money... a bird in hand worth two in the bush?

A mission like this would primarily be about flying skill, fuel management, and searching for allies to hide behind. I don't think we have anything like that right now (as of 0.10.8).

**Comment by tibetiroka** [comment]
> should Bounty Hunters be allowed to refuel in Republic space?

Yes. If you can refuel but they can't, it's pretty easy to get away from them. It would also greatly limit where they could appear, as any system more than a couple jumps from a pirate planet would be out of their reach.

Per the Navy not interfering, that's also a matter of balance. There are a lot of Navy ships in Republic space; it wouldn't be worth it for a bounty hunter fleet to go into their space, as they would get killed almost certainly.

Btw, we do have missions where you have to dodge projectiles, and that comes with a bit of fuel management as well.

**Reply by opusforlife2** [reply]
Not with the sort of Tom and Jerry chase where Tom has to worry about the dog (Butch? Brutus?) as well. They are just straight up enemies chasing you, every time.

It's pretty obvious that changing Bounty Hunter rep with the Republic would be completely game-breaking, which is why I pivoted to thinking of a different kind of mission style here.

I was doing the Ssil Vida mission. Here, for example, Bounty Hunters could play smart. You have to exit human space somehow, and they could have buddies on the lookout in the outer edge, covering more area, so that the mark doesn't escape. In every system you enter, you have to choose between pushing forward, potentially landing yourself in a death trap, or go back to safety, where the threat of Republic ships makes the hunters either stay away, or quickly escape the system if they were on your tail. Since the mission is on a timer, it would really amp up the difficulty. Right now, with a Peregrine or Albatross, this is just a filler mission.

**Comment by tuuktuuk** [comment]
This can be fixed by telling somewhere that the bounty hunters always pay a bribe when landing on a Republic planet (or FW planet). The player does the same when the Free Worlds and Republic are at war with each other: As long as I pay the bribe, I'm welcome everywhere. And, I think, there is more reason for republic planets to deny an enemy combatant a landing even for a bribe than to allow a bounty hunter to do the same. I'd imagine they don't need to pay anything on Hai planets, though. Or, on Pirate worlds.

The bounty hunters might also have a kind of a flatrate for bribes, like trucks in the Russia do: They collectively pay a monthly sum to the police and vehicles registered as being in that pool never get fines for overweight in that province.

Or, maybe the Republic spaceport doormen have a motive to keep the bribes low for bounty hunters? Maybe something about their presence in the spaceport is beneficial to a spaceport doorman?

**Reply by opusforlife2** [reply]
I can see bribes making sense, but only for some regions of space, such as the Dirt Belt and the South (pre-revolution). No way would the stuffy rich folks of the Paradise Planets allow any such disturbances to their perfect lives. Similarly for the Core, where the Syndicate rules. Crime of any kind is horrible for business, plain and simple.

And the Free Worlds have an in-lore reason to hate Pirates, and by extension Bounty Hunters. They're the very reason the Militias are formed, and later the FW.

**Reply by tuuktuuk** [reply]
On the Paradise Planets there are definitely inhabitants who made their wealth through crime. They might be influential enough to organize a scheme where all kinds of surprising things can be made possible.

I don't have a very unbribable image of the Republic in general, anyway. FW a bit more, but then again their history is such that a lot of people don't necessary follow their ideas (or laws) all that precisely.

I cannot find excuses for the Deep, though. It seems quite organized and fair. (But also: Not very important in the game all in all. It might not be a bad idea if the bounty hunters were unable to fuel up on planets over there)

**Reply by opusforlife2** [reply]
> On the Paradise Planets there are definitely inhabitants who made their wealth through crime. 

Well, yes, but the purpose was to be able to live in paradise for the rest of their lives. Kinda hard to do that when you're an aging drug lord a la Heisenberg. 🤭
